workitem/workitem_revision_repository.go

```diff
@@ -35,8 +35,9 @@ type GormRevisionRepository struct {
 // Create stores a new revision for the given work item.
 func (r *GormRevisionRepository) Create(ctx context.Context, modifierID uuid.UUID, revisionType RevisionType, workitem WorkItem) error {
 	log.Info(nil, map[string]interface{}{
-		"pkg":              "workitem",
-		"ModifierIdentity": modifierID,
+		"pkg":               "workitem",
+		"modifier_identity": modifierID,
+		"revision_type":     revisionType,
```

Comment:
@kwk snake case for variables apply to all letters as lower case, and words separated by underscores. I don't understand anything else for variables. 

The examples aforementioned are camel case